packs/signalr/sensors/signalr_hub_sensor.py

```diff
@@ -0,0 +1,43 @@
+from st2reactor.sensor.base import Sensor
+from signalr import Connection
+
+__all__ = [
+    'SignalRHubSensor'
+]
+
+
+class SignalRHubSensor(Sensor):
+    def __init__(self, sensor_service, config=None):
+        super(SignalRHubSensor, self).__init__(sensor_service=sensor_service,
+                                               config=config)
+        self._logger = self._sensor_service.get_logger(__name__)
+        self.url = config['hub_url']
+        self.hub_name = config['hub_name']
+        self._trigger_ref = 'signalr.message_received'
+        self._hub = None
+        self.connection = None
+        self.session = None
+
+    def setup(self):
+        self.connection = Connection(self.url, self.session)
+        # start a connection
+        self.connection.start()
+        # add a handler to process notifications to the connection
+        self.connection.handlers += \
+            lambda data: self._logger.log(
```

Comment:
I am pretty sure there is no `log` method. You want `debug` or `info` I'd think.